### PR TITLE
Make FIRST transparency grants fancy

### PR DESCRIPTION
**Pull request by InternetRamen**
Closes #3494 
This PR
 - Adds `transaction--first`
 - Styles disbursements from Transparency Fund (1803)

**Best reviewed by**
 - styles (colors, font-weight)
 
image

### CODE REVIEW

File: app/models/disbursement.rb [hunk 1]
```diff
@@ -85,6 +85,13 @@ class Disbursement < ApplicationRecord
       css_class: "transaction--icy",
       icon: 'freeze',
       qualifier: ->(d) { d.source_event_id == EventMappingEngine::EventIds::WINTER_HARDWARE_WONDERLAND_GRANT_FUND }
+    },
+    first_transparency_grant: {
+      title: 'FIRST® Transparency grant',
+      memo: "🤖 FIRST® Transparency Grant",
+      css_class: "transaction--first",
+      icon: 'idea',
```
Comment (garyhtou) [inline app/models/disbursement.rb]: i believe we might have a robot icon named "sam". both work well tho, so up to you

**Review by garyhtou — COMMENTED** [review]
code-wise this is great! im concerned about having red be the main color because it could be mistaken as a negative transaction. could we try some other kind of color gradient?

**Comment by maxwofford** [comment]
Thoughts on renaming FIRST => FRC for variable and class names? I know it's technically for FRC/FTC/FLL, but I think this would help us avoid class names like `transaction--first`, which could be confusing without context (ie. is this the first transaction in a list?)

**Comment by maxwofford** [comment]
For colors, here's an example of something with similar colors to FIRST's branding: https://github.com/hackclub/bank/blob/5b5f0c08e92bcd991d39ab7553c7bac955b7ebff/app/assets/stylesheets/components/_promos.scss#L3

You can see it live at https://bank.hackclub.com/hq/promotions#pvsa. Specifically, try it in light and dark mode.

File: app/models/disbursement.rb (hunk 1 above)
Comment (InternetRamen) [reply app/models/disbursement.rb]: I used the idea icon bc I couldn't find the robot one haha

**Comment by InternetRamen** [comment]
> For colors, here's an example of something with similar colors to FIRST's branding:
> 
> https://github.com/hackclub/bank/blob/5b5f0c08e92bcd991d39ab7553c7bac955b7ebff/app/assets/stylesheets/components/_promos.scss#L3
> 
> You can see it live at https://bank.hackclub.com/hq/promotions#pvsa. Specifically, try it in light and dark mode.

@maxwofford I updated it so it looks like this
chrome_RAjKBmkQOx
I tried making it 1/3 for each FIRST color, but it doesn't seem to render properly
image
Any ideas?

**Comment by garyhtou** [comment]
Maybe we can tile their Icon as the background. I'm not sure if this would be in line with their [branding guidelines](https://www.firstinspires.org/sites/default/files/uploads/resource_library/brand/first-brand-guidelines-2020.pdf).

https://www.firstinspires.org/brand

**Comment by InternetRamen** [comment]
Hmm, I'll give it a try

**Comment by garyhtou** [comment]
i'd recommend designing this in something like Figma first so that we can see if the tiled background will work out. e.g. making sure there's enough contrast between the background and the text

**Comment by InternetRamen** [comment]
Screenshot 2023-02-17 1 50 42 PM
This is what I've got, I think we can use the logo

**Comment by InternetRamen** [comment]
Hey guys sorry for the delay, I've been busy with school but I'll push a commit tmr

**Comment by maxwofford** [comment]
No sweat @InternetRamen! Iove how that tiled background looks. Here's just a quick hand-done one, but i think if you do this with transparency and maybe .3 to .6 opacity it'd have a similar effect & fit into the bank theming well
amaziograph_5FB89CDD-0783-4282-9984-CBF06D26DEBD amaziograph

**Comment by InternetRamen** [comment]
> No sweat @InternetRamen! Iove how that tiled background looks. Here's just a quick hand-done one, but i think if you do this with transparency and maybe .3 to .6 opacity it'd have a similar effect & fit into the bank theming well amaziograph_5FB89CDD-0783-4282-9984-CBF06D26DEBD amaziograph
image
image

Just added it, but I'm having some issues having the background-repeat lineup and keep the same size for the icons. Any ideas?

**Comment by maxwofford** [comment]
See the feature previews for examples of its usage! https://github.com/hackclub/bank/blob/ee16f35af253b0d78ccfa6b14205f77ac6c99016/app/assets/stylesheets/components/_features.scss#L63-L71

**Comment by InternetRamen** [comment]
chrome_rP52m02Bw9
This PR is good to go

File: app/assets/stylesheets/components/_transactions.scss [hunk 2]
```diff
@@ -77,6 +105,9 @@
     .transaction--icy & {
       fill: $white;
     }
+    .transaction--first & {
```
Comment (garyhtou) [inline app/assets/stylesheets/components/_transactions.scss]: i think we can get rid of this

**Review by garyhtou — APPROVED** [review]
good to merge! just address that comment first!

**Review by maxwofford — APPROVED** [review]
Good to go! Merge when ready and don't forget to post about it

**Comment by maxwofford** [comment]
Merging with confirmation from @InternetRamen in the Slack

**Pull request merged**